Tools/Scripts/libraries/webkitbugspy/webkitbugspy/tracker.py

```diff
@@ -29,7 +29,7 @@
 
 
 class Tracker(object):
-    REFERENCE_RE = re.compile(r'((https|http|rdar|radar)://[^\s><,\'\"}{\]\[)(]*[^\s><,\'\"}{\]\[)(\.\?])')
+    REFERENCE_RE = re.compile(r'^<?((https|http|rdar|radar)://[^\s><,\'\"}{\]\[)(]*[^\s><,\'\"}{\]\[)(\.\?])>?')
```

Comment:
Pretty sure this needs the `re.MULTILINE` flag to match the WKBI.

We probably want to make sure we have tests both with `<>` and for multiple lines.

Reply:
Good catch! Added flag and tests for multiline - `<>` tests already exist